hidapi/hidapi.h

```diff
@@ -103,10 +105,11 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
+		/** A structure to hold the version numbers. */
 		struct hid_api_version {
-			int major;
-			int minor;
-			int patch;
+			int major; ///< major version number
+			int minor; ///< minor version number
+			int patch; ///< patch version number
```

Comment:
as per agreement - we use only `/** c-style comments */` - for consisteny

Reply:
Oh, I couldn't find any code style guideline, but I didn't look very hard...

Reply:
done. The `<` is required as part of doxygen trailing docstring syntax.

Reply:
> couldn't find any code style guideline

unfortunately those aren't written anywhere
just the fact of consistency